File: C#/ConsoleApp2/NewDirectory1/Person.cs
namespace ConsoleApp2.NewDirectory1;

public class Person
{
    private string _firstName;
    private string _lastName;
    private int _age;
    
    public Person(string firstName, string lastName, int age)
    {
        _firstName = firstName; 
        _lastName = lastName;
        _age = age;
        
    }

    public string FirstName
    {
        get { return _firstName; }
        set { _firstName = value; }
        
    }

    public string LastName
    {
        get { return _lastName; }
        set { _lastName = value; }
        
    }

    public int Age
    {
        get { return _age; }
        set { _age = value; }
    }
}

File: Java/ArrayClassExercise/src/Person.java
import java.util.Objects;

public class Person implements Comparable<Person>{
    private String firstName;
    private String lastName;
    private String pesel;

    public Person(String firstName, String lastName, String pesel) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.pesel = pesel;
    }


    @Override
    public String toString() {
        return "Person{" +
                "firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                ", pesel='" + pesel + '\'' +
                '}';
    }

    @Override
    public int compareTo(Person o) {
        if(o==null){
            return 0;
        }
        return this.firstName.compareTo(o.firstName);
    }
}
